packages/client/hmi-client/src/utils/facets.ts

```diff
@@ -175,6 +187,19 @@ export const getFacets = (results: SearchResults[], resultType: ResourceType | s
 	return facets;
 };
 
+export const getFacetNameFormatter: Function = (resultType: string, key: string) => {
+	if (resultType === ResourceType.XDD && key === GITHUB_URL) {
+		return function format(label: string) {
+			// @ts-ignore
+			// eslint-disable-next-line @typescript-eslint/no-unused-vars
+			const [url, ...rest] = label.split('github.com/');
```

Comment:
For future reference, you can use [`URL.pathname`](https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname#examples) to safely get this info.
```js
const repo = new URL(label).pathname;
```